app/views/mortgage_calculator/land_and_buildings_transaction_taxes/_bands_table.html.erb

```diff
@@ -1,18 +0,0 @@
-<table class="mortgagecalc__table stamp-duty__table">
-  <thead>
-    <tr>
-      <th class="mortgagecalc__table__price"><%= I18n.t("land_and_buildings_transaction_tax.table.property_price_header") %></th>
```

Comment:
Given you have now refactored this here
https://github.com/moneyadviceservice/mortgage_calculator/pull/333/files#diff-c03244dd20a04b073f19ce0ad3d866b7R7

you can now remove the `land_and_buildings_transaction_tax.table` translations